app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -66,6 +69,22 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
     }
 
     @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater menuInflater) {
+        menuInflater.inflate(R.menu.menu_contributors, menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
Maybe you should call `setHasOptionsMenu(true)` in `onCreate()` 💭 
https://github.com/DroidKaigi/conference-app-2017/blob/7f26024d1ee5c1985fbfc223af99c1e53fff7f89/app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java#L83